doc/src/tutorial/basic_operations.rst

```diff
@@ -155,6 +155,15 @@ user's discretion by setting the ``chop`` flag to True.
     >>> (one - 1).evalf(chop=True)
     0
 
+It is always suggested to use ``expr.evalf(subs=...)`` over,
+``expr.subs(...).evalf()`` or ``expr.evalf().subs(...)`` because it
+avoids the loss of significance caused due to naive substitution.
```

Comment:
There is already a bit about evalf(subs) above. I would just replace the example there with this one.